src/main/scala/is/hail/methods/LinearMixedRegression.scala

```diff
@@ -541,23 +445,77 @@ case class DiagLMM(
   TyTy: Double,
   useML: Boolean)
 
-object LMMConstants {
-  def apply(y: DenseVector[Double], C: DenseMatrix[Double],
-            S: DenseVector[Double], Ut: DenseMatrix[Double]): LMMConstants = {
-    val UtC = Ut * C
-    val Uty = Ut * y
+trait ScalarLMM
```

Comment:
Doesn't seem like there is any point in having the trait anymore. FullRankScalarLMM and LowRankScalarLmm no longer share an interface.